VideoToImage/videotoimage.py

```diff
@@ -0,0 +1,41 @@
+import botocore
+import subprocess as sp
+import os
+import boto3
+import datetime
+
+#No. of frames per second
+FRAMES = int(os.environ.get('FRAMESPERSEC'))
+
+def convertVideoToImage(srcKey, srcBucket):
+    s3 = boto3.resource('s3')
+    localFilename = '/tmp/{}'.format(os.path.basename(srcKey))
```

Comment:
the srckey is really a filepath. Are you sure this works if the key is something `video/foo/bar/video.mp4`